hub/core/dataset/view_entry.py

```diff
@@ -47,10 +47,12 @@ def load(self):
         return ds
 
     def optimize(self, unlink=True, progressbar=True):
-        """Optimizes the view by copying the required data.
+        """Optimizes the dataset view by copying and rechunking the required data. This is necessary to achieve fast streaming
+            speeds when training models using the dataset view. The optimization process will take some time, depending on
+            the size of the data.
 
         Args:
-            unlink (bool): Unlink linked tensors by copying data from the links to the view.
+            unlink (bool): If True, this unlinks linked tensors (if any) by copying data from the links to the view.
```

Comment:
mention that this won't be done for video tensors by default but this behaviour can be changed by setting hub.constants._UNLINK_VIDEOS = True